Server.MirForms/Account/AccountInfoForm.cs

```diff
@@ -91,6 +92,27 @@ public void RefreshInterface()
 
             List<AccountInfo> accounts = SMain.Envir.AccountList;
 
+            long totalgold = 0;
+            foreach (var account in accounts)
+            {
+                if (!account.AdminAccount && !account.Banned)
+                    totalgold += account.Gold;
+                ServerGoldTextBox.Text = totalgold.ToString();
+            }
+            CultureInfo culture = new CultureInfo("en-GB");
+            string formattedgold = totalgold.ToString("N0", culture);
+            ServerGoldTextBox.Text = formattedgold;
+
+            long totalcredit = 0;
+            foreach (var account in accounts)
+            {
```

Comment:
instead of looping through accounts, use linq Sum - its quicker and cleaner

accounts.Where(x => !x.AdminAccount && !x.Banned).Sum(s => s.Credit)